content/en-us/reference/engine/classes/Player.yaml

```diff
@@ -221,16 +221,16 @@ properties:
   - name: Player.Character
     summary: |
       A `Class.Model` controlled by the player that contains a `Class.Humanoid`,
-      body parts, scripts and other objects.
+      body parts, scripts, and other objects.
     description: |
       The Character property contains a reference to a `Class.Model` containing
-      a `Class.Humanoid`, body parts, scripts and other objects required for
+      a `Class.Humanoid`, body parts, scripts, and other objects required for
       simulating the player's avatar in-game. The model is parented to the
       `Class.Workspace`, but may be moved. It is automatically loaded when
       `Class.Players.CharacterAutoLoads` is true, but can be manually loaded
       otherwise using `Class.Player:LoadCharacter()`.
```

Comment:
```suggestion
      `Class.Workspace` but it may be moved. It is automatically loaded when
      `Class.Players.CharacterAutoLoads` is true and it can be manually loaded
      otherwise using `Class.Player:LoadCharacter()`.
```